README.md

```diff
@@ -10,17 +10,22 @@ It's very simple to create a single running MongoDB instance. All examples are u
 var Server = require('mongodb-topology-manager').Server;
 // Create new instance
 var server = new Server('binary', {
-  dbpath: f('%s/db', __dirname)
+  dbpath: './db'
 });
 
-// Perform discovery
-var result = yield server.discover();
-// Purge the directory
-yield server.purge();
-// Start process
-yield server.start();
-// Stop the process
-yield server.stop();
+const init = async () => {
+  // Perform discovery
+  var result = await server.discover();
+  // Purge the directory
+  await server.purge();
+  // Start process
+  await server.start();
+  // Stop the process
+  await server.stop();
+}
+
+// start the server
+init()
```

Comment:
Use semicolons consistently please

Reply:
oops. sorry about the semicolons. havent used them in a while :sweat_smile: . will do if @mbroadst approves

Reply:
@khubo LGTM! Please do as @vkarpov15 asks, but I'm on board with async-ifiying the documentation.